applications/dashboard/js/dashboard.js

```diff
@@ -625,6 +625,7 @@ var DashboardModal = (function() {
             if (this.settings.modalType !== 'noheader' && $title.length !== 0) {
                 title = $title.html();
                 $title.remove();
```

Comment:
If the title is in the .header-block, do we really need to remove the title separately?